lib/sync-DataSetModel.js

```diff
@@ -310,26 +316,31 @@ var self = {
   createDatasetClient: function (dataset_id, query_params, meta_data, cb) {
     self.getDataset(dataset_id, function (err, dataset) {
       if (err) return err;
-      var clientHash = dataset_id + "_" + self.getClientHash(query_params, meta_data);
-      var datasetClient = dataset.clients[clientHash];
-      if (!datasetClient) {
-        datasetClient = {
-          id: clientHash,
-          datasetId: dataset_id,
-          created: new Date().getTime(),
-          lastAccessed: new Date().getTime(),
-          queryParams: query_params,
-          metaData: meta_data,
-          syncRunning: false,
-          syncPending: true,
-          syncActive: true,
-          pendingCallbacks: [],
-          instances: {}
-        };
-        dataset.clients[clientHash] = datasetClient;
-      }
-      syncUtil.doLog(dataset_id, 'verbose', 'createDatasetClient :: ' + util.inspect(datasetClient));
-      return cb(null, datasetClient);
+      //var clientHash = dataset_id + "_" + self.getClientHash(query_params, meta_data);
+      //console.log('getDataset callback - Sync : ' + clientHash);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ACommentedCode)